aiidalab_widgets_base/computational_resources.py

```diff
@@ -45,7 +45,7 @@ class ComputationalResourcesWidget(ipw.VBox):
     codes = traitlets.Dict(allow_none=True)
     allow_hidden_codes = traitlets.Bool(False)
     allow_disabled_computers = traitlets.Bool(False)
-    input_plugin = traitlets.Unicode(allow_none=True)
+    default_calc_job_plugin = traitlets.Unicode(allow_none=True)
```

Comment:
Why are we changing this traitlet? This imo unnecessarily breaks backward compatibility and makes an extra work for AiiDAlab apps' migration.

Reply:
Okay, I think I see your point. Only mapping the interface when loaded from the database.

Reply:
Hi @danielhollas, I gave it a try and think it is better to all change to `default_calc_job_plugin `. Since `orm.InstalledCode` only receive `default_calc_job_plugin` as parameter, I think it is better to keep the widget API match with this.

Reply:
I think those traits are supposed to mimic the aiida attributes of a `Code` object. So I would prefer that we change them to be compatible with the latest updates of AiiDA core.